tests/unit/stage/test_serialize_pipeline_lock.py

```diff
@@ -144,11 +144,31 @@ def test_lock_outs_order(dvc, typ):
     )
 
 
-def test_dump_appropriate_checksums(dvc):
+def test_dump_nondefault_hash(dvc):
     stage = create_stage(
         PipelineStage, dvc, deps=["s3://dvc-temp/file"], **kwargs
     )
-    stage.deps[0].info = {"etag": "is-it-etag", "md5": "or-md5?"}
+    stage.deps[0].info = {"md5": "value"}
+    assert to_single_stage_lockfile(stage) == OrderedDict(
+        [
+            ("cmd", "command"),
+            (
+                "deps",
+                [
+                    OrderedDict(
+                        [("path", "s3://dvc-temp/file"), ("md5", "value")]
+                    )
+                ],
+            ),
+        ]
+    )
+
+
+def test_dump_multiple_hashes(dvc):
```

Comment:
We don't use this actively right now, but this is a result of us treating loading `out.info` as a leftover of unparsed values, which needs symmetry on dumping as well.